dev.zed.Zed.yaml

```diff
@@ -0,0 +1,105 @@
+# dev.zed.Zed.yaml
+app-id: dev.zed.Zed
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: zed
+separate-locales: false
+finish-args:
+  # Grant access to Direct Rendering Infrastructure (DRI) for GPU acceleration
+  - --device=dri
+
+  # Allow access to the user's home directory for reading and writing files
+  - --filesystem=home
+
+  # Allow shared memory access for inter-process communication
+  - --share=ipc
+
+  # Enable network access
+  - --share=network
+
+  # Provide access to X11 display server as a fallback
+  - --socket=fallback-x11
+
+  # Enable access to PulseAudio sound server
+  - --socket=pulseaudio
+
+  # Enable access to Wayland display server
+  - --socket=wayland
+
+  # Enable access to Canonical AppMenu
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=com.canonical.AppMenu.Registrar.*
+
+  # Enable access to D-Bus session bus (--socket=session-bus is not supported by Flathub's build system)
+  - --system-talk-name=org.freedesktop.DBus
+  - --talk-name=org.freedesktop.DBus
```

Comment:
This is granted by default. Why is `session-bus` needed?

Reply:
`session-bus` is mentioned because at the moment the authors of Zed don't know exactly all of the DBus calls so it looked like the sensible toggle until other comments pointed out that we needed to further limit permissions